### DISCUSSION TITLE
Prometheus V.0.2 UI

### DISCUSSION THREAD

**Discussion by Brohammer5** [prometheus-lua/Prometheus#52] [Ideas]
Hello, 
I think we should make Prometheus have a ui in v.0.2
Dicuss some designs that you think it should have.

**Comment by levno-710** [comment]
I am in the process of creating an UI, however it will probably not be finished for Prometheus v0.2

**Comment by NoppePay** [comment]
Ok, we can encrypt our files very nicely, but what should we do if we want to decrypt the encrypted files again later?

**Comment by levno-710** [comment]
You will need to keep the original source code

**Reply by NoppePay** [reply]
While giving the scripts I encrypted, I write a note about who it belongs to, but now someone has distributed my script, how can I find out who it is? If I send the file to you, can you decrypt it?
